addons/account/tests/test_account_payment_duplicate.py

```diff
@@ -0,0 +1,166 @@
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestAccountPaymentDuplicateMoves(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.receivable = cls.company_data['default_account_receivable']
+        cls.payable = cls.company_data['default_account_payable']
+        cls.bank_journal = cls.company_data['default_journal_bank']
+        cls.payment_in = cls.env['account.payment'].create({
+            'amount': 50.0,
+            'payment_type': 'inbound',
+            'partner_id': cls.partner_a.id,
+            'destination_account_id': cls.receivable.id,
+        })
+        cls.payment_out = cls.env['account.payment'].create({
```

Comment:
checked quickly to find a helper, but couldn't find one, maybe could be nice to check more deeply